features/poller/monitors/core/pom.xml

```diff
@@ -21,7 +21,9 @@
             <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
             <Bundle-Version>${project.version}</Bundle-Version>
-            <Export-Package>org.opennms.netmgt.poller.monitors.*;version="${project.version}"</Export-Package>
+            <Import-Package>com.novell.ldap.*;resolution:=optional,*</Import-Package>
+            <Private-Package>org.opennms.netmgt.poller.monitors.*;version="${project.version}";org.opennms.netmgt.poller.*;version="${project.version}"</Private-Package>
```

Comment:
Hi Seth,

When I removed the org.opennms.netmgt.poller.*;version="${project.version}". I get error shown below. However when I removed org.opennms.netmgt.poller.monitors.*;version="${project.version}" and installed, it worked. Can I remove poller.monitors instead?

karaf@root> features:install opennms-poller-monitors-core
Error executing command: Can't install feature opennms-poller-monitors-core/0.0.0: 	
Could not start bundle mvn:org.opennms.features.poller.monitors/org.opennms.features.poller.monitors.core/18.0.0-SNAPSHOT in feature(s) opennms-poller-monitors-core-18.0.0-SNAPSHOT: Unresolved constraint in bundle org.opennms.features.poller.monitors.core [121]: Unable to resolve 121.0: missing requirement [121.0] osgi.wiring.package; (&(osgi.wiring.package=org.opennms.netmgt.poller)(version>=18.0.0)(!(version>=19.0.0)))
karaf@root>